homeassistant/components/kraken/sensor.py

```diff
@@ -216,7 +216,7 @@ def available(self):
         return self._available and self.coordinator.last_update_success
 
     @property
-    def device_info(self) -> dict:
+    def device_info(self):
```

Comment:
This should return `DeviceInfo`.

https://github.com/home-assistant/core/blob/3200b0150af6c275de229cf22ce3bf9ab9e36be5/homeassistant/components/gios/air_quality.py#L125-L133